flutter_local_notifications/lib/src/platform_specifics/darwin/notification_attachment.dart

```diff
@@ -18,4 +19,25 @@ class DarwinNotificationAttachment {
   /// When left empty, the platform's native APIs will generate a unique
   /// identifier
   final String? identifier;
+
+  /// Should the attachment be considered for the notification thumbnail?
+  final bool? hideThumbnail;
+
+  /// The clipping rectangle for the thumbnail image.
+  final DarwinNotificationAttachmentThumbnailClippingRect? thumbnailClippingRect;
+}
+
+class DarwinNotificationAttachmentThumbnailClippingRect {
```

Comment:
- Can you add public API docs for these? This should've picked up by the linter
- Are you able to add an example that makes use of this?

Reply:
Both done!